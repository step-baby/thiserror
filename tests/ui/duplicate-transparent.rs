use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
#[display(transparent)]
#[display(transparent)]
pub struct Error(anyhow::Error);

fn main() {}
