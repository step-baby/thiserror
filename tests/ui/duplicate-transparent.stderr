error: duplicate #[display(transparent)] attribute
 --> tests/ui/duplicate-transparent.rs:5:1
  |
5 | #[display(transparent)]
  | ^^^^^^^^^^^^^^^^^^^^^
